tests/desktop/test_users_account.py

```diff
@@ -83,16 +83,13 @@ def test_hide_email_checkbox_works(self, mozwebqa):
         Assert.true(home_page.header.is_user_logged_in)
 
         view_profile_page = home_page.header.click_view_profile()
-        initial_state = view_profile_page.is_email_field_present
 
         edit_profile_page = home_page.header.click_edit_profile()
-        edit_profile_page.change_hide_email_state()
+        edit_profile_page.hide_email_address()
         edit_profile_page.click_update_account()
-
         view_profile_page = home_page.header.click_view_profile()
-        final_state = view_profile_page.is_email_field_present
 
-        Assert.not_equal(initial_state, final_state, 'The initial and final states are the same. The profile change failed.')
+        Assert.equal(view_profile_page.is_email_field_present, False, 'The email address is not hidden')
```

Comment:
If we're just asserting if something is True or False we don't need to use `Assert.equal`. We should either use `assert condition`, `assert not condition`, `Assert.true(condition)` or `Assert.false(condition)`. Preference to `assert ...` as we're moving away from unittestzero.